pkg/typesystem/typesystem.go

```diff
@@ -585,6 +586,34 @@ func (t *TypeSystem) TTUCanFastPath(objectType, tuplesetRelation, computedRelati
 	return true
 }
 
+func (t *TypeSystem) PathExists(user, relation, object string) (bool, error) {
+	userType, _, userRelation := tuple.ToUserParts(user)
+	userTypeRelation := userType
+	if userRelation != "" {
+		// this is an userset
+		userTypeRelation = tuple.ToObjectRelationString(userType, userRelation)
+	}
+	fromNode, err := t.authorizationModelGraph.GetNodeByLabel(userTypeRelation)
+	if err != nil {
+		return false, err
```

Comment:
If the userType doesn't exist in the model, where will the Check request fail, here or in some other method? I'm hoping it won't reach here and it will fail somewhere else

Reply:
Yes, we should never hit this.  However, from a function perspective, we still need to do validation.

Reply:
agreed. Can you verify whether the request fails earlier if the userType doesn't exist?